processing/src/main/java/io/druid/query/extraction/BucketExtractionFn.java

```diff
@@ -56,8 +57,13 @@ public double getOffset()
   }
 
   @Override
-  public String apply(Object value)
+  @Nullable
+  public String apply(@Nullable Object value)
   {
+    if (Objects.isNull(value)) {
```

Comment:
Objects.isNull methods exists for being used as lambda, direct use is just an obscure way to write value != null